dbcon/joblist/lbidlist.cpp

```diff
@@ -918,6 +968,16 @@ bool LBIDList::GetMinMax<int64_t>(int64_t& min, int64_t& max, int64_t& seq, int6
                                   const std::vector<struct BRM::EMEntry>* pEMEntries,
                                   execplan::CalpontSystemCatalog::ColDataType colDataType);
 
+template
+bool LBIDList::GetMinMax<__int128>(__int128* min, __int128* max, int64_t* seq,
```

Comment:
I believe the code doesn't compile but just out of curiosity did you try to compile it w/o this explicit template specialization?

Reply:
The reason why I'm asking is that there is only one version of GetMinMax and that is templated. linker might ask for an explicit template to link against it though.

Reply:
Linker complains if I remove this template instantiation. Reason for this is the client of GetMinMax() is tuple-bps.cpp, which is a separate translation unit.